src/BlazorUI/Demo/Client/Bit.BlazorUI.Demo.Client.Core/Pages/Components/Navs/NavBar/BitNavBarDemo.razor.scss

```
@@ -1,34 +1,138 @@
-﻿@import '../../../../Styles/abstracts/_media-queries.scss';
+@import '../../../../Styles/abstracts/_functions.scss';
```

Stylelint (17.14.0)

[error] Unexpected leading underscore in imported partial name (scss/load-no-partial-leading-underscore)

(scss/load-no-partial-leading-underscore)